src/script_interface/walberla/LBFluid.cpp

```diff
@@ -139,6 +139,12 @@ void LBFluidGPU::make_instance(VariantMap const &params) {
   auto const visc = get_value<double>(params, "kinematic_viscosity");
   auto const dens = get_value<double>(params, "density");
   auto const precision = get_value<bool>(params, "single_precision");
+  auto const blocks_per_mpi_rank = get_value_or<Utils::Vector3i>(
+      params, "blocks_per_mpi_rank", Utils::Vector3i{{1, 1, 1}});
```

Comment:
Here and elsewhere, do we really need a default value, considering we already provide a default value in the python class?

Reply:
I added default value for the python class `LatticeWalberla`. And `blocks_per_mpi_rank` in `LBFluid` is settle by `get_value<Utils::Vector3i>(m_lattice->get_parameter("blocks_per_mpi_rank"))`.

Reply:
Therefore, we don't need to provide a default value in C++ code, so I changed `get_value_or` to `get_value`.